### DISCUSSION TITLE
Select Field - group options (optgroup)

### DISCUSSION THREAD

**Discussion by bratvanov** [payloadcms/payload#9397] [Feature Requests & Ideas]
Select fields in HTML use the [optgroup](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup) element to visually group options.

React Select's [Group and GroupHeading components](https://react-select.com/components) replicate this functionality and are utilized in Payload's [Relationship Field](https://payloadcms.com/docs/fields/relationship) to display `relationTo` slugs as group headings.

It would be a valuable enhancement if [Payload's Select](https://payloadcms.com/docs/fields/select#config-options) field also supported option grouping. While this likely wouldn't impact stored data, it would greatly improve UI/UX by organizing large lists of options more effectively.

Similarly, the [Payload Form Builder](https://payloadcms.com/docs/plugins/form-builder#select) plugin should also support option groups in select fields.

**Comment by seunghyukchoe** [comment]
## Cycle 1 complete — backlog cleared, zero open PRs

`main` is at `97b56ce` with **1,557 tests** passing (1,494 at cycle open).

### Merged this cycle

| PR | What landed |
|---|---|
| #79 | Typed refusal for non-orthogonal saved footprint poses — a **verified untyped crash** through the placement boundary |
| #57 | Atomic route-bundle preview (all-or-nothing multi-net plans, ADR-0066) |
| #58 | Bounded multilayer proposals, 2–8 signal layers (ADR-0068) |
| #59 | Route-aware placement scoring, opt-in and advisory (ADR-0067) |
| #81 | Repository professionalization: docs IA, ADR index, link checker |

Closed as superseded with evidence: #48, #49, #50 — each proven by byte-identical blobs or commit containment on main, each spun out into follow-up issues rather than dropped.

### What the review layer caught

Every substantive PR had a real defect the green suite did not: a schema-valid request that crashed untyped; an unrelated change that **silently reversed security decision SEC-113** (DRC output moved from DEVNULL to never-read files); a **silent break of two-layer candidate identity** while the ADR, ledger, changelog and PR body all claimed identity was preserved; and a benchmark measuring a different thing than it claimed (the compared candidate sets were disjoint, not re-ranked).

One finding is a negative result worth stating plainly: at the honest probe count, route-aware scoring produces a **worse** placement than the baseline (391 mm vs 359 mm total probe length), and all four "unrouted" probes turned out to be `off_grid` refusals rather than routability statements. That is recorded in B-082 and ADR-0067 rather than buried; whether it should gate integration is a maintainer call.

### Structural issues opened

#80 (no golden identity pins anywhere — how the identity break passed 1,506 tests), #82 (`check_ledgers.py` never inspects IDs), #83 (ADR/ledger IDs collide across concurrent branches — observed for real when two branches both claimed ADR-0066), plus #72/#74 (courtyard fidelity and ring nesting) and #75–#78 (live IPC hardening).

### Agentic hours

**Finished this cycle: ~26 agent-hours** — 5 deep PR reviews with local real-KiCad gates, 5 remediation passes, 1 professionalization pass, 4 merge-conflict resolutions including an ADR renumbering.

**Remaining to finish the roadmap: ~70–95 agent-hours.** M1 ~8–14, M2 ~15–26 (negotiated congestion and fill-aware routing are the big items), M3 ~8–14, M4 ~12–24, M5 ~13–30 (Rust/GPU/physics remains the most speculative). Three follow-up PRs are already in flight against that total.
